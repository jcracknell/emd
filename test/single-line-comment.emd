Paragraph
// single line comment
continues.

Paragraph// comment
continues.

Paragraph
// comment

Ends.

Paragraph //comment

Ends.
