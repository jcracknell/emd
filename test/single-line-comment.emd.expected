#! emd.cs.Conversion.StructuralRenderer, emd.cs
(Document
  (Paragraph
    (Text "Paragraph")
    (Space)
    (Text "continues.")
  )
  (Paragraph
    (Text "Paragraph")
    (Space)
    (Text "continues.")
  )
  (Paragraph
    (Text "Paragraph")
  )
  (Paragraph
    (Text "Ends.")
  )
  (Paragraph
    (Text "Paragraph")
  )
  (Paragraph
    (Text "Ends.")
  )
)
